scripts/unixbench/dependencies.txt

```diff
@@ -9,18 +10,21 @@ unixbench_order = 50
 # git = git clone using above giturl
 # pip = python pip utility
 # man = "manual"
-linpack-install = man
+git-install = man
+unixbench-install = man
 ### END - Dependency-specific installation method ###
 
 ### START - Tests ###
+git-configure = git --version
+unixbench-configure = ls -la 3RDPARTYDIR/byte-unixbench/UnixBench/Run
 ### END - Tests ###
 
 ### START - Dependency versions ###
 unixbench-ver = ANY
+git-ver = ANY
 ### END - Dependency versions ###
 
-### START - Dependency URLs ###
-### END - Dependency URLs ###
-
 ### START -  Dependency and method-specific command lines ###
+common-git-install-man = package_install git-all
+common-unixbench-install-man = git clone https://github.com/kdlucas/byte-unixbench.git
```

Comment:
I don't mean to be pedantic here, but this should be a fully-functional set of copy-and-paste commands. What happens here is that CloudBench spits out this command to the user in the case of `man` (manual) mode. Thus, this needs to include the correct path (a home directory, perhaps) where the clone should go, as well as the complete set of commands to configure, compile and install the binary in a location which precisely matches where the benchmark script that you have above is expecting it to be.